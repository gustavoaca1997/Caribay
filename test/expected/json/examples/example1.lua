return {
    tag = 'json',
    {
        -- Fulanito
        tag = 'value',
        {
            tag = 'object',
            {
                tag = 'pair',
                { tag = 'STRING', '"first name"' },
                {
                    tag = 'value',
                    { tag = 'STRING', '"Fulanito"' },
                }
            },{
                tag = 'pair',
                { tag = 'STRING', '"last name"' },
                {
                    tag = 'value',
                    { tag = 'STRING', '"Perez"' },
                }
            }, {
                tag = 'pair',
                { tag = 'STRING', '"age"' },
                {
                    tag = 'value',
                    { tag = 'NUMBER', '13' },
                },
            }, {
                tag = 'pair',
                { tag = 'STRING', '"single"' },
                {
                    tag = 'value',
                    { tag = 'BOOLEAN', 'true' },
                },
            },
            {
                tag = 'pair',
                { tag = 'STRING', '"parents"' },
                {
                    tag = 'value',
                    {
                        tag = 'array',
                        {
                            -- Fulana
                            tag = 'value',
                            {
                                tag = 'object',
                                {
                                    tag = 'pair',
                                    { tag = 'STRING', '"first name"' },
                                    {
                                        tag = 'value',
                                        { tag = 'STRING', '"Fulana"' },
                                    }
                                },{
                                    tag = 'pair',
                                    { tag = 'STRING', '"last name"' },
                                    {
                                        tag = 'value',
                                        { tag = 'STRING', '"Lopez"' },
                                    }
                                }, {
                                    tag = 'pair',
                                    { tag = 'STRING', '"age"' },
                                    {
                                        tag = 'value',
                                        { tag = 'NUMBER', '46' },
                                    },
                                }, {
                                    tag = 'pair',
                                    { tag = 'STRING', '"single"' },
                                    {
                                        tag = 'value',
                                        { tag = 'BOOLEAN', 'false' },
                                    },
                                }, {
                                    tag = 'pair',
                                    { tag = 'STRING', '"parents"' },
                                    {
                                        tag = 'value',
                                        { tag = 'array', },
                                    },
                                },
                            }
                        },
                        {
                            -- Fulano
                            tag = 'value',
                            {
                                tag = 'object',
                                {
                                    tag = 'pair',
                                    { tag = 'STRING', '"first name"' },
                                    {
                                        tag = 'value',
                                        { tag = 'STRING', '"Fulano"' },
                                    }
                                },{
                                    tag = 'pair',
                                    { tag = 'STRING', '"last name"' },
                                    {
                                        tag = 'value',
                                        { tag = 'STRING', '"Perez"' },
                                    }
                                }, {
                                    tag = 'pair',
                                    { tag = 'STRING', '"age"' },
                                    {
                                        tag = 'value',
                                        { tag = 'NUMBER', '48' },
                                    },
                                }, {
                                    tag = 'pair',
                                    { tag = 'STRING', '"single"' },
                                    {
                                        tag = 'value',
                                        { tag = 'BOOLEAN', 'false' },
                                    },
                                },
                                {
                                    tag = 'pair',
                                    { tag = 'STRING', '"parents"' },
                                    {
                                        tag = 'value',
                                        {
                                            tag = 'array',
                                            {
                                                -- Mengano
                                                tag = 'value',
                                                {
                                                    tag = 'object',
                                                    {
                                                        tag = 'pair',
                                                        { tag = 'STRING', '"first name"' },
                                                        {
                                                            tag = 'value',
                                                            { tag = 'STRING', '"Mengano"' },
                                                        }
                                                    },{
                                                        tag = 'pair',
                                                        { tag = 'STRING', '"last name"' },
                                                        {
                                                            tag = 'value',
                                                            { tag = 'STRING', '"Perez"' },
                                                        }
                                                    }, {
                                                        tag = 'pair',
                                                        { tag = 'STRING', '"age"' },
                                                        {
                                                            tag = 'value',
                                                            { tag = 'NUMBER', '75' },
                                                        },
                                                    }, {
                                                        tag = 'pair',
                                                        { tag = 'STRING', '"single"' },
                                                        {
                                                            tag = 'value',
                                                            { tag = 'BOOLEAN', 'true' },
                                                        },
                                                    }, {
                                                        tag = 'pair',
                                                        { tag = 'STRING', '"parents"' },
                                                        {
                                                            tag = 'value',
                                                            { tag = 'array', },
                                                        },
                                                    },
                                                }
                                            },
                                        }
                                    }
                                }
                            }
                        },
                    }
                }
            }
        }
    }
}