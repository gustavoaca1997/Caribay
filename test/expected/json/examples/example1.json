{
    "first name": "Fulanito",
    "last name": "Perez",
    "age": 13,
    "single": true,
    "parents": [
        {
            "first name": "Fulana",
            "last name": "Lopez",
            "age": 46,
            "single": false,
            "parents": []
        },
        {
            "first name": "Fulano",
            "last name": "Perez",
            "age": 48,
            "single": false,
            "parents": [
                {
                    "first name": "Mengano",
                    "last name": "Perez",
                    "age": 75,
                    "single": true,
                    "parents": []
                }
            ]
        }
    ]
}